cmd/node/flags.go

```diff
@@ -507,26 +517,66 @@ func getWorkingDir(workingDir string, log logger.Logger) string {
 }
 
 func applyCompatibleConfigs(log logger.Logger, configs *config.Configs) error {
-	importDbFlags := configs.ImportDbConfig
-	importDbFlags.ImportDbNoSigCheckFlag = importDbFlags.ImportDbNoSigCheckFlag && importDbFlags.IsImportDBMode
```

Comment:
thinking again, we need to keep this in a form or another as we might specify the no sig check flag but not enable the import-db main flag, yielding in a very very bad configuration

Reply:
Great. Now the node will signal a critical error and stop. :+1: